global.css

```diff
@@ -0,0 +1,10 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Your absolute positioned navbar with 100% with was causing a slight width overflow (which causes a horizontal scroll on the webpage)

This is how you would fix that and make sure other elements don't cause it.